src/lib/fetchPersistedQuery.ts

```diff
@@ -0,0 +1,22 @@
+import queryMap from "../data/complete.queryMap.json"
+import { info, error } from "./loggers"
+import { RequestHandler } from "express"
+
+export const fetchPersistedQuery: RequestHandler = (req, res, next) => {
```

Comment:
I see, clever, so it will set the query (as though you had sent it in the body) after a quick check before the graphql middleware

Reply:
Neat, makes sense!